stroyprombeton/tests/tests_selenium_admin.py

```diff
@@ -80,11 +74,18 @@ def setUp(self):
             parent_id=self.children_category_id).first().id)
         self.tree_product_id = str(Product.objects.filter(
             category_id=self.deep_children_category_id).first().id)
+        self.change_products_url = self.live_server_url + reverse(
+            'admin:stroyprombeton_productpage_changelist')
         self.sign_in()
 
+    def tearDown(self):
+        # Clear this to avoid checking of UI states.
+        # It reduces time of execution and mitigates error happenings.
+        self.browser.execute_script('localStorage.clear();')
+
     @property
     def first_h1(self):
-        return self.browser.find_elements_by_tag_name('h1')[1].text
+        return self.wait.until(EC.presence_of_all_elements_located((By.TAG_NAME, 'h1')))[1].text
```

Comment:
`presence_of_element_located` will be enough, i suppose. But maybe it's not true

Reply:
an admin page has several h1, so we have to use `presence_of_all_elements_located`